libr/cons/utf8.c

```diff
@@ -255,7 +255,11 @@ R_API int r_cons_is_utf8() {
 }
 #else
 R_API int r_cons_is_utf8() {
+#if UNICODE && __WINDOWS__
+	return 1;
+#else
 	return 0;
+#endif
```

Comment:
By the way, I think we should determine this property on the fly.
See https://docs.microsoft.com/en-us/windows/console/getconsoleoutputcp and https://docs.microsoft.com/en-us/windows/desktop/api/winnls/nf-winnls-enumsystemcodepagesa